jest.config.js

```diff
@@ -0,0 +1,19 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+const { defaults: tsjPreset } = require('ts-jest/presets');
+
+module.exports = {
+  preset: 'ts-jest',
+  transform: {
+    // For transformations with .ts and .tsx files.
+    ...tsjPreset.transform,
+    // For transformations with .js and .jsx files.
+    '^.+\\.(js|jsx)$': 'babel-jest',
```

Comment:
Is this necessary? I thought ts-jest would handle regular JS files also and not require Babel? Could also come back to this later. I'd generally love to live in a Babel-free future (or at least where it's handled by something else and I don't have to touch it).

Reply:
Yes, if there are tests written in js or jsx Jest will fail without this. It is documented in integrating Jest and TS. I would also prefer a babel free future. Unfortunately getting Jest to work with TS with WP Scripts involves using babel.